projects/igniteui-angular/src/lib/snackbar/snackbar.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="igx-snackbar" *ngIf="isVisible" (@slideInOut.start)="snackbarAnimationStarted($event)" (@slideInOut.done)="snackbarAnimationDone($event)"
     [@slideInOut]="isVisible">
-    <span class="igx-snackbar__message" [@fadeInOut]="isVisible">{{ message }}</span>
+    <span class="igx-snackbar__message" [@fadeInOut]="isVisible" *ngIf="message">{{ message }}</span>
+    <ng-content></ng-content>
```

Comment:
Shouldn't the `ng-content` be wrapped by a container with the `igx-snackbar__message` class? or like a content class, which provides default alignment and positioning?

Reply:
Not sure about that ... We may want to have some igx-snackbar__content class, but assigning the message class to some custom content may not be the best idea in all scenarios.

Reply:
@gedinakova @mtsvyatkova You should sync this with @simeonoff

Reply:
@kdinev I am with Konstantin on this one. Regardless of whether you want to use the message input attribute, or simply place any content between the opening and closing brackets(preferred), the _message_ should be wrapped in an element with class igx-snackbar__message.

If you have time, you can also do away with the whole wrapping div in the template, hostbinding the 'igx-snackbar' class. Animations will have to be reimplemented, though.

Reply:
The message is now wrapped in a container with the custom content with the igx-snackbar__message class. 
The removal of the wrapping div element will be researched and probably included in another PR.